eeps/eep-0064.md

```diff
@@ -1,176 +1,480 @@
+quoting all backslashes.
+
+We could also do like Elixir and choose both, but then we have
+to implement [Sigils][] or something like that before getting
+triple-quoted strings that are useful enough.
+
+So if we have to choose one format, it should be the Verbatim one,
+since that is the more general format, also in code;
+where you simply would want to define a string for some purpose,
+impossible to foresee.
+
+This does not close the door on [Sigils][].  We can state that
+we have merely chosen that the default for triple-quoted strings
+is verbatim, and for normal strings it is escaped.  Although
+it is a bit annoying that we then do not have the same defaults
+as [Elixir][], there are other more annoying subtle differences
+between strings in the languages.  See [Comparison with Elixir][].
+
+Because the strings are verbatim, and we want to use them
+to define any string in code, we cannot have the limitation
+that there always is a newline at the end.  Therefore the
+last newline (the newline on the line that precedes
+the string ending line) should be stripped.  It is easy
+to add a newline if an ending newline is needed.
+
+[Elixir][] does not strip the last newline and dodges this problem
+by escaping newlines.  If you do not want the last newline you put
+a backslash last on the last line.  This does not work in their
+verbatim strings, though.  So you have to choose between either
+verbatim with ending newline or having to escape backslashes.
+
+A slightly annoying consequence of verbatim strings with a fixed
+end marker there is no possibility to create a string containing
+such an end marker.  Therefore this EEP, as a possible extension,
+suggests allowing not only triple-quoted strings,
+but 3-or-more-quoted strings.  Then a start and end marker
+can be chosen that is not part of the string,
+and any string can be created.  This is a not so pretty solution
+for a very rare corner case.
+
+### Triple-Quoted String Scanner Token
+
+A triple-quoted string must be a token that the scanner recognizes
+as a string, which makes it suitable for a documentation attribute
+value term.  It starts and ends with three double quotes: `"""`.
+
+Double quotes, `"`, are chosen because normal Erlang strings use them
+and this is just a new variant.  Since double quotes are used
+a triple-quoted string shall, as a normal string,
+produce a list of code-points.
+
+It would be more convenient if a triple-quoted string produced
+an UTF-8 binary, but that would be a surprising feature for double quotes,
+and the documentation build process can work around this by converting
+the code-point list (`string()`) into the needed binary chunk.
+
+In source code a triple-quoted string is valid within a binary,
+so producing a Unicode binary is reasonably straightforward:
+
+    X = <<"""
+        Line 1
+        Line 2
+        """/utf8>>
+
+The 2 + 7 characters overhead is not exhausting since we are targeting
```

Comment:
I know this is obvious to most people but I do not follow where these characters 2 + 7 mean or come from.
Not sure if you can explain offline or here, so that the EEP has all content

Reply:
The 2 is `<<` and the 7 is `/utf8>>` which precede/follow the triple quotes.